buildpipeline/Core-Setup-Publish.json

```diff
@@ -488,8 +488,25 @@
     "PB_ToolsRoot": {
       "value": "$(Build.SourcesDirectory)\\tools"
     },
-    "SymbolCatalogCertificateId": {
-      "value": "400"
+    "PB_OptionalToolSource": {
+      "value": "https://devdiv.pkgs.visualstudio.com/_packaging/dotnet-core-internal-tooling/nuget/v3/index.json "
```

Comment:
@dagood don't we already handle the optional tools download? If not how do we currently get the embed index tool?

Reply:
Right now it's only in `Core-Setup-Signing-Validation.json`, I believe the changes in this file are just copying it over. (Extra space is also there.)